### ISSUE TITLE
Could you share pre-trained weight?

### ISSUE THREAD

**Issue by ReuiYan-Lin** [Adamdad/keras-YOLOv3-mobilenet#34]
Thanks for the work.
Could you share pre-trained weight of mobilenet | 320*320 | VOC 2007 ?
I'm using train_Mobilenet.py to train VOC2007 dataset, but I get the mAP 32%.
I find that the train loss and val loss are around 13 and cannot get lower.
I have tried increasing the epoch but the performance only improve a little.
Could you share pre-trained weight  or some information of hyperparameters(learning rate、epoch etc.)
